papermill/tests/test_execute.py

```diff
@@ -2,22 +2,25 @@
 import os
 import pytest
 import shutil
-import sys
 import tempfile
 import unittest
 
 from functools import partial
 
+import six
+
 import nbformat
 
 from ..api import read_notebook
 from ..execute import execute_notebook, log_outputs, _translate_type_python, _translate_type_r
 from ..exceptions import PapermillExecutionError
 from . import get_notebook_path, RedirectOutput
 
-python_2 = sys.version_info[0] == 2
-PYTHON = 'python2' if python_2 else 'python3'
-execute_notebook = partial(execute_notebook, kernel_name=PYTHON)
+
+if six.PY2:
+    execute_notebook = partial(execute_notebook, kernel_name='python2')
+else:
+    execute_notebook = partial(execute_notebook, kernel_name='python3')
```

Comment:
simpler as `execute_notebook = partial(execute_notebook, kernel_name='python3' if six.PY2 else 'python2')`

Reply:
Can we split the difference on this one? I wouldn't ever expect a ternary inside an argument like that and it makes the line super long (as in >79chars which is my religion ;) )

Reply:
That 80 char limit is super irritating in modern coding environments (especially with 4 space python :P). If you really don't like the ternary then at least keep it a bit more DRY and make a kernal_name variable which captures the six logic and only repeat `execute_notebook = partial(execute_notebook, kernel_name=kernel_name)` once

Reply:
Sound alright? If you strong disagree I'm still ok with merging as it's better than it was regardless.

Reply:
Done.

Next you are going to tell me some of your friends use tabs :p